src/shared/sleep-config.c

```diff
@@ -206,6 +207,12 @@ int battery_is_low(void) {
          * or Normal in case ACPI is not working properly. In case of no battery
          * 0 will be returned and system will be suspended for 1st cycle then hibernated */
 
+        r = on_ac_power();
+        if (r < 0)
+                log_debug_errno(r, "Failed to check if the system is running on AC, ignoring: %m");
```

Comment:
maybe include ", assuming is not", to indicate clearly how this cases is treated now.

Reply:
or maybe say "assuming it might not be"